cdap-ui/app/features/hydratorplusplus/services/non-store-pipeline-error-factory.js

```diff
@@ -203,7 +203,7 @@ let hasValidSchedule = (importConfig, GLOBALS) => {
 };
 let hasValidInstance = (importConfig, GLOBALS) => {
   let isRealtimePipeline = importConfig.artifact.name === GLOBALS.etlRealtime;
-  return !isRealtimePipeline? true: importConfig.config.instance;
+  return !isRealtimePipeline? true: importConfig.config.instances;
```

Comment:
I am pretty sure we have a whole bunch of places where we need to do this. We should do this as part of a separate PR post 3.4. This is just a linting issue that shouldn't affect this change.